model/src/cfd.rs

```diff
@@ -1396,14 +1396,12 @@ impl Cfd {
         };
 
         match event {
```

Comment:
Arguably we could also say that force close should be a warning, because it is not a happy path scenario. What do you think @klochowicz @bonomat ?

Reply:
Yes, maybe a good idea. I think it won't hurt to make it stand out a bit more than happy path.

Reply:
I like this idea! we try to be alerted to it on grafana, this will make it stand out more indeed.